helmfile.d/charts/rclone/values.yaml

```
@@ -37,12 +37,15 @@ crypt:
 # - Providers translate into the configured provider
 targets:
   - destinationName: dn
+    destinationPath: ""
```

Would setting it to `/` for "root" make sense or does that not work as I think?
```suggestion
    destinationPath: /
```

---

I think that would work as well. But I don't think I have tested it and I'm sure that "" works. So I suggest just keeping it as is.

---

Okay, let's keep the empty string.